src/client/app/helpers/theme.ts

```diff
@@ -51,6 +51,14 @@ export function applyTheme(host: string): Promise<boolean> {
                 });
                 break;
 
+            case HostType.OUTLOOK:
+                (require as any).ensure([], (require) => {
+                    require(['style-loader!raw-loader!postcss-loader!sass-loader!../../assets/styles/themes/outlook.scss'], () => {
```

Comment:
This isn't actually possible. Due to the way that require.ensure works, we have to do it this way.